namespaces/kubernetes.yml

```diff
@@ -94,6 +94,131 @@ namespace:
         description: >
           The provider associated with this Kubernetes object
 
+  - name: event
+    type: group
+    description: >
+      The kubernetes event obtained from kubernetes master API
+      The event is already JSON object and as whole nested under kubernetes field
+      This description should loosely follow 'type Event' in https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go
+    fields:
```

Comment:
We should try to reduce duplication as much as possible. I will have notes further down about this.

These mappings define how Elasticsearch parses, stores, and indexes these values, so defining the mapping ahead of time can have significant savings in storage and search.

Creating a nested, hierarchical data structure in fluentd costs more than a flat structure, so we should be aware of that, but we don't want an entirely flat structure either.